projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -1097,6 +1110,20 @@ function wpcom_launchpad_is_videopress_launch_disabled() {
 	return ! wpcom_is_checklist_task_complete( 'videopress_upload' );
 }
 
+/**
+ * Determines whether or not the blog launch task is enabled
+ *
+ * @return boolean True if blog launch task is enabled
+ */
+function wpcom_launchpad_is_blog_launched_disabled() {
```

Comment:
I think it would be a bit clearer if the name included `task` so we know what this function is doing/checking.
```suggestion
function wpcom_launchpad_is_blog_launched_task_disabled() {
```